enterprise/frontend/src/embedding-sdk/components/private/AppInitializeController.tsx

```diff
@@ -28,7 +31,10 @@ export const AppInitializeController = ({
         isLoggedIn,
       }}
     >
-      <SdkContentWrapper font={config.font ?? DEFAULT_FONT}>
+      <SdkContentWrapper
+        id={EMBEDDING_SDK_ROOT_ELEMENT_ID}
```

Comment:
This will only work with one provider, I think we already thought of having multiple providers on one page even at the start to let customers work around some initial limitations we could have (like being able to set some settings/plugins for the whole provider).
Should we use useId or similar and pass it down on a context/redux?

Reply:
Actually I think they would just all use the same div but it should work, getElementById doesn't crash if he finds more than 1 element with the same id and even though ids should be unique I don't think there is any practical consequence